app/scripts/templates/test_template.mustache

```diff
@@ -37,10 +37,12 @@
     <div class="input-row password-row">
       <input type="password" class="password" id="password" pattern=".{8,}" required />
       <input id="show-password" type="checkbox" class="show-password" aria-controls="password" data-novalue />
+      <div class="input-help input-help-focused input-help-signup">Must be at least 8 characters</div>
     </div>
     <div class="input-row password-row">
       <input type="password" class="password" id="vpassword" pattern=".{8,}" required />
       <input id="show-password" type="checkbox" class="show-password" aria-controls="vpassword" data-novalue />
+      <div class="input-help input-help-complete-password">Must be at least 8 characters</div>
```

Comment:
Ignore this comment. We are changing the way we word things related to passwords, and I am fixing this in another PR anyway.